collectors/bzimport/convertors.py

```diff
@@ -595,6 +595,34 @@ def major_incident_state(self):
 
         return valid_pairs.get(flags_from_bz, Flaw.FlawMajorIncident.INVALID)
 
+    @cached_property
+    def requires_summary(self):
+        """
+        A requires_summary state created from the requires_doc_text flag.
+        """
+        # Default values
+        status = ""
+        setter = ""
+
+        # Sets values from Bugzilla
+        for flag in self.flags:
+            if flag["name"] == "requires_doc_text":
+                setter = flag["setter"]
+                status = flag["status"]
+
+        # this combination is set automatically by BZ when Doc Text is added
+        if setter == "bugzilla@redhat.com" and status == "+":
+            return Flaw.FlawRequiresSummary.REQUESTED
+
+        pairs = {
```

Comment:
I suggest using `mapping` instead if `pairs`. I think it's more clear of what the dict is. `pairs` seem like a leftover from the dual-flag `major_incident_state()`'s `valid_pairs`.